data/human/free worlds 4 epilogue.txt

```diff
@@ -208,13 +208,13 @@ mission "FW Epilogue: Danforth"
 mission "FW Epilogue: Edward"
 	landing
 	name `Weapon Testing`
-	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
+	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edwards feedback on its performance.`
```

Comment:
I'm fairly confident that his name is "Edward" and that the one place where he is currently called "Edwards" (the first line of the conversation associated with this mission) is actually what is incorrect.
```suggestion
	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
```